extensions/stats/plugin.cc

```diff
@@ -542,6 +542,10 @@ std::optional<uint32_t> PluginRootContext::addIntExpression(
 
 bool PluginRootContext::onDone() {
   cleanupExpressions();
+  if (!request_queue_.empty()) {
+    LOG_CRITICAL(absl::StrCat("Request queue is not empty, dropping requests: ",
```

Comment:
Will this crash envoy? I don't think we want to crash envoy because of stats drop. Should be at warning level I suppose?

Also do we need to do anything for SD plugin?

Reply:
It shouldn't crash, it's just a log level per https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/blob/7afb39d868a973caa6216a535c24e37fb666b6f3/proxy_wasm_api.h#L88. 

Critical because it should never happen, so it's basically an assert.

I looked over SD, and it seems to be exporting onDone. Let me know if you can think of a situation when it doesn't export.